src/components/Stages/AddStage/index.tsx

```diff
@@ -170,6 +171,25 @@ function AddStage() {
                   })}
               </div>
 
+              <div className={`${style.inputDiv} ${style.targetInputMargin}`}>
```

Comment:
The erc20 token is already added when he created the campaign. That is what is expected to be sent to the stage.

Reply:
Where are we adding the amount when we create the campaign?

Reply:
We haven't deposited any amount yet, this task is to add a button for it to be added after the campaign is created.

Reply:
Done. Added a modal in single stage page to commit erc